helm/cloudnative-pg/templates/image-catalog.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: ClusterImageCatalog
+metadata:
+  name: gs-postgresql
+  annotations:
+    "helm.sh/hook": "post-install,post-upgrade"
+  labels:
+    {{- include "cloudnative-pg.labels" . | nindent 4 }}
+spec:
+  images:
+    - major: 16
+      image: quay.io/giantswarm/postgresql-cnpg:16.2
```

Comment:
Why quay? and can we template the registry?

Reply:
The image isn't being crsynced for some reason. The registry will definitely be templated once that's resolved

Reply:
Actually, it seems it is synced to aliyun and azure, just not Docker :thinking:

Reply:
This turned out to be a quirk of crsync. I've updated the catalog to template in the registry